ReactTask/src/components/MainPatientsAppointment/MainPatientsAppointment.js

```diff
@@ -9,11 +10,18 @@ import {
   StylBoxPatientContent,
   StylBoxRowPadding,
   StylTitlePatients,
+  StylBoxPatientsList,
 } from './../';
 import heartImgSvg from './../../img/icons/icon-heart.svg';
-import dataPatient from './../../db/dbProfilePatient.json';
+import { getPatients } from '../../redux/patient/patientOperations';
 
 const MainPatientsAppointment = ({ switchContent }) => {
+  const [appointments, setAppointments] = useState([]);
+
+  useEffect(() => {
+    getPatients(0, 5).then((response) => setAppointments(response.data));
```

Comment:
If i am right it  call magic number. I point these two params in function. You can check how i did this. I mean this is bad practice to put this parametrs from component

Reply:
Fixed